low-level/sspl-ll.spec

```diff
@@ -94,6 +94,8 @@ SSPL_DIR=/opt/seagate/%{product_family}/sspl
     ln -sf $SSPL_DIR/low-level/resource_health_view /usr/bin/resource_health_view
     ln -sf $SSPL_DIR/low-level/sspl_bundle_generate /usr/bin/sspl_bundle_generate
     ln -sf $SSPL_DIR/low-level/manifest_support_bundle /usr/bin/manifest_support_bundle
+    ln -sf $SSPL_DIR/low-level/framework $SSPL_DIR/low-level/files/opt/seagate/sspl/setup/health_provider/
+    ln -sf $SSPL_DIR/low-level/files/opt/seagate/sspl/setup/health_provider $SSPL_DIR/bin/health_provider
```

Comment:
health_provider is renamed to resource_map, need that change here too

Reply:
Done.